drivers/can/can_stm32.c

```diff
@@ -503,9 +517,10 @@ static int can_stm32_init(const struct device *dev)
 	data->state_change_cb = NULL;
 	data->state_change_cb_data = NULL;
 
-	data->filter_usage = (1ULL << CAN_STM32_MAX_NUM_FILTERS) - 1ULL;
-	(void)memset(data->rx_cb, 0, sizeof(data->rx_cb));
-	(void)memset(data->cb_arg, 0, sizeof(data->cb_arg));
+	(void)memset(data->rx_cb_std, 0, sizeof(data->rx_cb_std));
+	(void)memset(data->cb_arg_std, 0, sizeof(data->cb_arg_std));
+	(void)memset(data->rx_cb_ext, 0, sizeof(data->rx_cb_ext));
+	(void)memset(data->cb_arg_ext, 0, sizeof(data->cb_arg_ext));
 
```

Comment:
We can get rid of all these `memset()` and `NULL` initializers since `data` is `static` and therefore zeroed.

Reply:
Good point. Done.